components/brave_wallet_ui/components/shared/untrusted-image/untrusted-image.tsx

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
```

Comment:
Looks like the changes that were made in this file have been reverted. Can you double check them?